server/controllers/boardController.js

```diff
@@ -4,18 +4,18 @@ import {} from '../models/issueModel';
 
 const getBoardDetails = async (req, res) => {
   const id = parseInt(req.params.id, 10);
-  if (Number.isNaN(id)) {
+  if (Number.isNaN(parseInt(id, 10))) {
```

Comment:
Why are you using two times parseInt on a variable?

Reply:
😅 Yup, by mistake of merged code.